up42/workflow.py

```diff
@@ -95,7 +108,7 @@ def get_workflow_tasks(self, basic: bool = False) -> Union[List, Dict]:
         logger.info(f"Got {len(tasks)} tasks/blocks in workflow {self.workflow_id}.")
 
         if basic:
-            return {task["name"]: task["id"] for task in tasks}
+            return {task["name"]: task["blockVersionTag"] for task in tasks}
```

Comment:
Nice! Can we construct a new workflow from this dictionary?

Reply:
How do you mean, as a functionality to clone a workflow?

Reply:
Yes - intuitively I would say it this returns a dict, I should be able to use this dict to add tasks to a workflow.

Reply:
Hm yes, we could have this format as the default option to add blocks with a specific version, would be nice:

So simple format to get the newest block version:
`workflow.add_workflow_tasks(["blockname1", "blockname2"])`

or to specify the block version:
`workflow.add_workflow_tasks({"blockname1":"version1", "blockname2":"version2"})`

The only problem is that the output of `get_workflow_tasks(basic=True)` and `workflow_tasks` also contains the additional `:1` as the workflow task structure indication. Guess we either remove that here or check/handle it in `add_workflow_tasks`.

Reply:
Hm ok, seems a bit more complicated, maybe we do it some other time, since we want to deploy.